quelpa.el

```diff
@@ -882,7 +894,16 @@ Return a cons cell whose `car' is the root and whose `cdr' is the repository."
         (when (file-exists-p dir)
           (delete-directory dir t))
         (quelpa-build--princ-checkout repo dir)
-        (quelpa-build--run-process nil "git" "clone" repo dir)))
+        (cond
+         ((integerp quelpa-git-clone-depth)
```

Comment:
Since the variable must be either an integer or nil, it's not necessary to test it with `integerp`.